polyfills/Object.prototype.toString/polyfill.js

```diff
@@ -0,0 +1,6 @@
+var Object_prototype_toString = Object.prototype.toString
+Object.prototype.toString = function() {
+    if (this === null) return "[object Null]";
```

Comment:
It's faster and clearer for a caller to explicitly check null and undefined anyways before using `Object#toString`